src/enhancedScatterChartSettingsModel.ts

```diff
@@ -19,27 +21,27 @@ export enum DisplayUnitsType {
     Thousands = 1000,
     Millions = 1000000,
     Billions = 1000000000,
-    Triilions = 1000000000000
+    Trillions = 1000000000000
```

Comment:
replace units with default, see sankey for reference
https://github.com/microsoft/powerbi-visuals-sankey/blob/dev/capabilities.json#L123
https://github.com/microsoft/powerbi-visuals-sankey/blob/dev/src/settings.ts#L122

Reply:
resolved here https://github.com/microsoft/powerbi-visuals-enhancedscatter/pull/102/commits/c1a5957a59de58b5e6668c6158ee386eaf1053b2